### ISSUE TITLE
Fix color contrast on email newsletter Subscribe button

### ISSUE THREAD

**Issue by sugaroverflow** [drupaldiversity/drupaldiversity.github.io#40]
> #fff on #39B5F2 doesn't meet WCAG AA or AAA, let's change it.

http://webaim.org/resources/contrastchecker/?fcolor=ffffff&bcolor=39B5F2

Moved over from https://github.com/drupaldiversity/administration/issues/85.

**Comment by sugaroverflow** [comment]
+ Related from @cleverington 
Apparently our version of Blue only goes with Black and pretty much just black.

I think we need to explore a third color. I was thinking something either purplish or pinkish.

**Comment by laughnan** [comment]
Closing this issue. We have migrated to Drupal 8 at https://www.drupaldiversity.com. Any further work/issues are tracked on Drupal.org at: https://www.drupal.org/project/issues/ddi_contrib
